test/test_profile.py

```diff
@@ -52,26 +52,23 @@ def test_profile_error(default_params, task_manager_creator):
         task_manager_creator((manager_params, browser_params[:1]))
 
 
-@pytest.mark.skip(reason="proxy no longer supported, need to update")
-def test_profile_saved_when_launch_crashes(default_params, task_manager_creator):
+def test_profile_saved_when_launch_crashes(
+    monkeypatch, default_params, task_manager_creator
+):
     manager_params, browser_params = default_params
     manager_params.num_browsers = 1
     browser_params[0].profile_archive_dir = (
         manager_params.data_directory / "browser_profile"
     )
-    browser_params[0].proxy = True
-    browser_params[0].save_content = "script"
     manager, _ = task_manager_creator((manager_params, browser_params[:1]))
     manager.get(BASE_TEST_URL)
 
-    # Kill the LevelDBAggregator
-    # This will cause the proxy launch to crash
-    manager.ldb_status_queue.put("DIE")
+    # This will cause browser restarts to fail
+    monkeypatch.setenv("FIREFOX_BINARY", "/tmp/NOTREAL")
```

Comment:
This is an absolutely genius way to introduce a crash at startup. Well done!

Reply:
Thank you! An alternative would be to unset one of [these two preferences](https://github.com/mozilla/OpenWPM/blob/625d81460b25d68846de36ddeeab23c9731bbda6/openwpm/deploy_browsers/configure_firefox.py#L251L252). This would result in the error observed in issue https://github.com/mozilla/OpenWPM/issues/606, because the extension won't be able to start.